resources/views/vendor/twill/partials/head.blade.php

```diff
@@ -0,0 +1,33 @@
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no" />
+<meta http-equiv="X-UA-Compatible" content="IE=Edge">
+<meta name="robots" content="noindex,nofollow" />
+
+<title>{{ config('app.name') }} {{ config('twill.admin_app_title_suffix') }}</title>
+
+<link rel="icon" href="favicon-red.ico">
```

Comment:
This is the only difference between this override and Twill's original template.

Reply:
Can you wrap this line in comments to make it clear this is the only different? So when we upgrade we know what to replace? You'll see an example of this in layouts/form.blade.php. Something like:

```
{{-- WEB-2788: Use red favicon in CMS --}}
<link rel="icon" href="favicon-red.ico">
{{-- WEB-2788: EOF --}}
```